www/js/app.js

```diff
@@ -379,6 +379,7 @@ document.getElementById('btnTop').addEventListener('click', function (event) {
 document.getElementById('btnHome').addEventListener('click', function (event) {
     // Highlight the selected section in the navbar
     event.preventDefault();
+    this.blur();
```

Comment:
Is it necessary to do this? It prevents keyboard users from keeping focus on the button, and so prevents them toggling with the space key. This reduces accessibility.